src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/ArrayInitHandler.java

```diff
@@ -74,7 +75,16 @@ protected DetailAST getLeftCurly() {
 
     @Override
     protected IndentLevel curlyIndent() {
-        final IndentLevel level = new IndentLevel(getIndent(), getBraceAdjustment());
+        int offset = 0;
+
+        final DetailAST lcurly = getLeftCurly();
+
+        if (isOnStartOfLine(lcurly)
+            && !TokenUtil.isOfType(lcurly.getParent(), TokenTypes.ARRAY_INIT)) {
```

Comment:
If you are just comparing one type, then why don't you just use an equal sign? I am not seeing a need to use the util method here.

Reply:
Corrected